cfgmgr/intfmgr.cpp

```diff
@@ -115,6 +144,16 @@ void IntfMgr::setIntfIp(const string &alias, const string &opCmd,
     int ret = swss::exec(cmd.str(), res);
     if (ret)
     {
+        /*
+            ipv6 setting on the iface might still haven't been enabled by SDK.
```

Comment:
This comment refers to Nvidia SDK. It is better not to mention that in common code.

Reply:
Will remove